eng/common/CIBuild.cmd

```diff
@@ -1,4 +1,3 @@
 @echo off
-powershell -ExecutionPolicy ByPass -command "& """%~dp0Build.ps1""" -restore -build -projects """%~dp0..\GenerateAnalyzerRulesets\GenerateAnalyzerRulesets.csproj""" %*"
```

Comment:
I added this step in previous PR https://github.com/dotnet/roslyn-analyzers/pull/1718 to enable generation of common rulesets that are packaged into analyzer packages. Are we building this project somewhere else now?

Reply:
Could you take a look again? I have integrated the ruleset generation to the build.

Reply:
Thanks, the change looks good to me, as long as we have validated the output NuGet packages contain the generated rulesets.